btcec/go.mod

```diff
@@ -1,16 +1,19 @@
 module github.com/btcsuite/btcd/btcec/v2
 
-go 1.17
+go 1.22
```

Comment:
Also, requiring Go 1.22 when it doesn't actually require it is bad for consumers of the dependency too.